pkg/ipfs/node.go

```diff
@@ -64,16 +64,7 @@ type Node struct {
 	apiAddresses []string
 }
 
-func NewNode(ctx context.Context, cm *system.CleanupManager) (*Node, error) {
```

Comment:
was only used by the IPFS downloader. The downloader now calls `NewNodeWithConfig` directly.